locations/spiders/gls-deu.py

```diff
@@ -0,0 +1,70 @@
+import scrapy
+import json
+import csv
+from locations.items import GeojsonPointItem
+
+HEADERS = {"X-Requested-With": "XMLHttpRequest"}
+STORELOCATOR = "https://api.gls-pakete.de/parcelshops?version=4&coordinates={},{}&distance=40"
+
+class GenspiderSpider(scrapy.Spider):
+    name = 'glsdeuSpider'
+    allowed_domains = ['https://www.gls-pakete.de/']
+    deList = []
+    usedLongLat = []
+
+    def start_requests(self):
+        searchable_point_files = [
+            "./locations/searchable_points/eu_centroids_40km_radius_country.csv",
+        ]
+
+        for point_file in searchable_point_files:
+            with open(point_file) as openFile:
+                results = csv.DictReader(openFile)
+                for result in results:
+                    if(result["country"] == "DE"):
+                        longitude = format(float(result["longitude"]), '.5f')
+                        latitude = format(float(result["latitude"]), '.5f')
```

Comment:
It is required to do it.

Reply:
Instead of doing the string formatting here, please do it as part of the URL template. Instead of `{},{}`, you can use `{:0.5},{:0.5}`.

```
>>> "{},{}".format(0.4123423, 0.853592)
'0.4123423,0.853592'
>>> "{:0.5},{:0.5}".format(0.4123423, 0.853592)
'0.41234,0.85359'
```